manifests/freshclam.pp

```diff
@@ -33,7 +45,10 @@
       enable     => $clamav::freshclam_service_enable,
       hasrestart => true,
       hasstatus  => true,
-      subscribe  => File['freshclam.conf'],
+      subscribe  => [
+        File['freshclam.conf'],
+        File['freshclam_sysconfig'],
+      ],
```

Comment:
This may fail if not on RedHat 7.
Can you instead do something like lines 55-57?

If both freshclam_service and freshclam_sysconfig then notify...